From 7fab702e9b52802ab411fa56ad17bac50590ecaf Mon Sep 17 00:00:00 2001
From: Michael Natterer <mitch@gimp.org>
Date: Sun, 01 Dec 2013 22:00:24 +0000
Subject: Bug 719662 - plug-in-borderaverage returns a garbage alpha value

Return a fully opaque color instead of a random-memory alpha value.

(cherry picked from commit 56cdf642fc005ff03da67107c7c64308b36fa3e5)
---
diff --git a/plug-ins/common/border-average.c b/plug-ins/common/border-average.c
index be265fb..8df76b3 100644
--- plug-ins/common/border-average.c
+++ plug-ins/common/border-average.c
@@ -301,7 +301,7 @@ borderaverage (GimpDrawable *drawable,
     }
 
   /* return the color */
-  gimp_rgb_set_uchar (result, r, g, b);
+  gimp_rgba_set_uchar (result, r, g, b, 255);
 
   g_free (buffer);
   g_free (cube);
--
cgit v0.9.2
